how-to/integrate-with-rss/client/src/integrations-shapes.ts

```diff
@@ -0,0 +1,162 @@
+import type {
+	CLIFilter,
+	HomeDispatchedSearchResult,
+	HomeSearchListenerResponse,
+	HomeSearchResponse,
+	HomeSearchResult
+} from "@openfin/workspace";
+import type { BrowserWindowModule, Page } from "@openfin/workspace-platform";
+import { Notification, NotificationOptions } from "@openfin/workspace/notifications";
+
+/**
+ * Integration manager provides environment methods and data.
+ */
+export interface IntegrationManager {
+	/**
+	 * The root url for the provider.
+	 */
+	rootUrl?: string;
+
+	/**
+	 * Launch a view in the workspace.
+	 * @param view The view to launch.
+	 * @param targetIdentity The optional target identity to launch with.
+	 * @param targetView The optional target view to launch with.
+	 * @returns The launched view.
+	 */
+	launchView?(
+		view: OpenFin.PlatformViewCreationOptions | string,
+		targetIdentity?: OpenFin.Identity,
+		targetView?: OpenFin.Identity
+	): Promise<OpenFin.View>;
+
+	/**
+	 * Launch a page in the workspace.
+	 * @param page The page to launch.
+	 * @param bounds The optional bounds for the page.
+	 * @returns The window created.
+	 */
+	launchPage?(page: Page, bounds?: OpenFin.Bounds): Promise<BrowserWindowModule>;
+
+	/**
+	 * Launch a snapshot.
+	 * @param snapshotUrl The snapshot url
+	 */
+	launchSnapshot?(snapshotUrl: string): Promise<OpenFin.Identity[]>;
+
+	/**
+	 * Open a url with the browser.
+	 * @param url The url to open.
+	 */
+	openUrl?(url: string): Promise<void>;
+
+	/**
+	 * Show the home UI.
+	 */
+	showHome?(): Promise<void>;
+
+	/**
+	 * Create a notification.
+	 */
+	createNotification?<T extends NotificationOptions>(options: T): Promise<Notification<T>>;
```

Comment:
I think we should add createNotification to the interface. Reason being the integration needs to reference notification api for event listeners so I'd rather not pollute with something that isn't launching an app/result.  Are we using show home?

Reply:
You mean *not* add?

Reply:
yep sorry not add.